src/patternfly/components/Button/examples/buttonStates.hbs

```diff
@@ -1,3 +1,58 @@
-{{#> button btnClass="pf-is-primary pf-has-focus"}}Focused{{/button}}
-{{#> button btnClass="pf-is-primary pf-is-active" btnAriaPressed="true"}}Active{{/button}}
-{{#> button btnClass="pf-is-primary pf-is-disabled" btnTabIndex="-1" btnAriaDisabled="true"}}Disabled{{/button}}
\ No newline at end of file
+<div class="pf-l-level">
+<div class="pf-l-stack">
+    {{#> button btnClass="pf-is-primary"}}
+        Primary
+    {{/button}}
+    {{#> button btnClass="pf-is-primary pf-is-focus"}}
+        Primary Focus
+    {{/button}}
+    {{#> button btnClass="pf-is-primary pf-is-active" btnAttributes="ariaPressed=\"true\""}}
+        Primary Active
+    {{/button}}
+    {{#> button btnClass="pf-is-primary pf-is-disabled" btnAttributes="disabled=\"true\" tabIndex=\"-1\" ariaDisabled=\"true\"" }}
+        Primary Disabled
+    {{/button}}
+</div>
+<div class="pf-l-stack">
+    {{#> button btnClass="pf-is-secondary"}}
+        Secondary
+    {{/button}}
+    {{#> button btnClass="pf-is-secondary pf-is-focus"}}
+        Secondary Focus
+    {{/button}}
+    {{#> button btnClass="pf-is-secondary pf-is-active" btnAttributes="ariaPressed=\"true\""}}
```

Comment:
I don't think this is the right case for using `aria-pressed`. According to the [description for aria-pressed](https://www.w3.org/TR/wai-aria-1.1/#aria-pressed), the presence of this attribute indicates that it's a toggle button, like the Mute button shown in this example: https://www.w3.org/TR/wai-aria-practices-1.1/examples/button/button.html

I'd suggest removing this attribute altogether, until we're ready to provide examples of toggle buttons.

Reply:
@jgiardino Again, I'm following what was in the existing documentation for buttons regarding the active state. Since we are manually setting `pf-is-active` here to demonstrate what the active state looks like, I read it that we should include `aria-pressed`. If I should remove it from the example, is the comment in the documentation still valid to include?

Reply:
I wouldn't say this statement is wrong to include: 

> An active state button will appear pressed when active. There’s no need to add a class to <button>s as they use a pseudo-class. However, you can still force the same active appearance with .pf-is-active (and include the aria-pressed="true" attribute) should you need to replicate the state programmatically.

My thinking is that if the attribute `aria-pressed` should only be used for toggle buttons, then we should only include it in examples where we are explicitly showing toggle buttons. 

It would probably be good to clarify the documentation so that it's more obvious what the purpose of the attribute is, and say something like:

> However, if you are using a button as a toggle, you can still force the same active appearance with .pf-is-active (and include the aria-pressed="true" attribute) should you need to replicate the state programmatically.